js.js

```diff
@@ -0,0 +1,124 @@
+console.log("hello world!");
+console.log("hallo wereld!");
+console.log("selam dunya!");
+
+console.log("I'm awesome");
+console.log('I\'m awesome');
+
+
+let x;
+console.log("the value of x is undefined");
+console.log(x);
+x = 5;
+console.log("the value of x is 5");
+console.log(x);
+
+
+let y = "string";
+console.log("the value of y is string");
+console.log(y);
+y = "new string";
+console.log("the value of y now is 'new string'");
+console.log(y);
+
+
+let z = 7.25;
+console.log(z);
+let a = Math.floor(z);
+console.log(a);
+let h;
+if (z>a){
+    h = z ;
+}else {
+    h = a;
+};
+console.log("the highest value is "+h);
+
+let friends = [];
+console.log("the value is array");
+console.log(friends);
+let myFavoriteAnimals = ["cat's","fish's","turtle's","bird's"];
+console.log(myFavoriteAnimals);
+myFavoriteAnimals[4] = "Daan's";
```

Comment:
Haha, this works! But a cleaner way to do this by using:

```
myFavoriteAnimals.push("Daan's");
```

Reply:
okay ,i only use this to give a certain position to "daan's"